helpers/examples.mk

```diff
@@ -0,0 +1,6 @@
+GOSS_VERSION := v0.3.6
+
+goss:
+	kubectl cp test/*.yaml $(GOSS_CONTAINER):/tmp/goss.yaml
+	kubectl exec $(GOSS_CONTAINER) -- sh -c "cd /tmp/ && curl -s -L https://github.com/aelsabbahy/goss/releases/download/$(GOSS_VERSION)/goss-linux-amd64 -o goss && chmod +rx ./goss && ./goss validate --color --format documentation"
```

Comment:
While fiddling with this, I had an idea to use `kubectl run` for testing the service if we need such a thing:

```
IMAGE := docker.elastic.co/elasticsearch/elasticsearch:6.5.4

goss:
	tar -C test -zc . | kubectl run -i --attach --generator=run-pod/v1 --rm --image $(IMAGE)  helm-test -- sh -c 'cd $(mktemp -d); curl -vOL https://github.com/aelsabbahy/goss/releases/download/v0.3.6/goss-linux-amd64; chmod +x ./goss-linux-amd64; tar -zxv;  ./goss-linux-amd64 validate --color --format documentation'
```

This copies the files using a tar pipe over stdin through kubectl. Some changes would need to be made, though, since everything is `localhost` and you would want to refer to `elasticsearch-master` as the hostname. The tests fail largely due to `localhost` and kernel/filesystem checks not passing since it's not running in the elasticsearch-master container.

Reply:
> While fiddling with this, I had an idea to use kubectl run for testing the service if we need such a thing

We could also test the service from within the same container too right? As in adding another `http` test which hits the service instead of localhost. Testing that the service works from another container wouldn't really add anything unless I'm missing something.